config/webpack.config.dev.js

```diff
@@ -108,7 +108,7 @@ module.exports = {
       // Process JS with Babel.
       {
         test: /\.(js|jsx)$/,
-        include: paths.appSrc,
+        include: [paths.appSrc, paths.rum],
```

Comment:
this should not be required since we by default ship es5 now, Please update to latest `@elastic/apm-rum` if its the case.